packages/map-template/src/helpers/GetLegendSortedFields.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Get the legend sorted fields.
+ *
+ * @param {object} kioskLocation
+ * @returns {Promise}
+ */
+export default function getLegendSortedFields(kioskLocation) {
```

Comment:
I'm wondering if this could be a Recoil Selector that depends on the kioskLocation atom.

Reply:
Hmm, I can't see how that would really work. We use the kioskLocation in many places, and the legend is only used in two places.

Reply:
I don't see how converting getLegendSortedFields to a Recoil Selector (which would also use the kioskLocation atom) could affect the use of the kioskLocation atom. Can you elaborate?

Reply:
Used Recoil Selector ✅